src/NodaTime/TimeZones/DateTimeZoneNotFoundException.cs

```diff
@@ -14,7 +14,7 @@ namespace NodaTime.TimeZones
     /// <remarks>
     /// This type only exists as <c>TimeZoneNotFoundException</c> doesn't exist in the Portable Class Library.
```

Comment:
Gah - yes, I only search for PCL. I'll search for "Portable Class Library" too. (It's still not in netstandard 1.x, but *is* in 2.x)